app/controllers/WhoAreYouAgentController.scala

```diff
@@ -46,18 +46,20 @@ class WhoAreYouAgentController @Inject() (
     extends FrontendBaseController
     with I18nSupport {
 
-  val form = formProvider()
+  val form           = formProvider()
+  private val logger = play.api.Logger(getClass)
 
   def onPageLoad(mode: Mode, draftId: DraftId): Action[AnyContent] =
     (identify andThen getData(draftId) andThen requireData) {
       implicit request =>
-        val _ = request.userAnswers.get(WhoAreYouAgentPage) match {
+        logger.info("WhoAreYouAgentController onPageLoad")
+
+        val preparedForm = request.userAnswers.get(WhoAreYouAgentPage) match {
```

Comment:
I don't think the scaffolding uses this, but there's a handy function for this: `WhoAreYouAgentPage.fill(form)`

Reply:
Refactored it